business_requirement_deliverable_report/README.rst

```diff
@@ -0,0 +1,145 @@
+.. figure:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================================
+Business Requirement Deliverable Report Module
+==============================================
+
+Introduction
+^^^^^^^^^^^^
+
+This module is part of a set ("Business Requirement").
```

Comment:
I think this generic description should be in the main module, but not copied on each submodule. You can put something like: `Refer to the documentation on the main module to know what is a business requirement`.